cmd/cvp-trigger/main.go

```diff
@@ -4,69 +4,165 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"sort"
+	"strings"
 
 	"github.com/ghodss/yaml"
+	"github.com/openshift/ci-tools/pkg/util"
 	"github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
```

Comment:
What problem is this solving?

Reply:
I leveraged this because it allows for an interface which improves testing in my opinion. We can provide a `memMapOs` object in lieu of writing to the testing system's file-system and have greater flexibility for things such as validation for existence of paths, etc.